config.toml

```diff
@@ -48,10 +48,10 @@ windows_dl_link = "https://downloads.redisinsight.redislabs.com/latest/redisinsi
 docker_command = "docker run -v redisinsight:/db -p 8001:8001 redislabs/redisinsight:latest"
 docker_db_volume_permission = "chown -R 1001 redisinsight"
 
-rdi_redis_gears_min_semantic_version = "1.2.5"
-rdi_redis_gears_min_version = "10205"
-rdi_redis_gears_current_semantic_version = "1.2.6"
-rdi_redis_gears_current_version = "10206"
+rdi_redis_gears_min_semantic_version = "1.2.7"
+rdi_redis_gears_min_version = "10207"
+rdi_redis_gears_current_semantic_version = "1.2.7"
+rdi_redis_gears_current_version = "10207"
 rdi_debezium_server_version = "2.1.1.Final"
```

Comment:
I thought we've already changed this to `2.3.0`, @galilev please check.

Reply:
✅

Reply:
2.3.0.Final